app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,64 @@
+class QuestionsController < ApplicationController
+
+  def index
+    @vote = Vote.new
+    @questions = Question.all
+  end
+
+  def recent
+    @questions = Question.order(created_at: :desc)
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = current_user.questions.new(question_params)
+    if @question.save
+      flash[:success] = "Question created successfully"
+      redirect_to questions_path
+    else
+      flash[:now] = "There was a problem"
+      render 'new'
+    end
+  end
+
+  def show
+    @question = Question.find(params[:id])
+    @answer = Answer.new
+    @answers = @question.answers.includes(:user)
```

Comment:
You could have `@question = Question.includes(answers: :user)` then iterate through @question.answers